### DISCUSSION TITLE
Welcome to W1000 bilingual discussions!

### DISCUSSION THREAD

**Discussion by ZsBT** [ZsBT/hass-w1000-portal#2] [Announcements]
## Hi!

Share your ideas, doubts, etc. Supported languages: English and Hungarian :)

**Comment by bankizsolt** [comment]
Kedves Zsombor!
Lehet valahogy javítani a január 13-14. i adatokon?

**Comment by ZsBT** [comment]
Szia Zsolt!

Bejáratott módszer nincs, de elvileg megoldható lenne hogy a teljes
visszamenő statisztika után hosszabb időre, pl 30 napra visszamenőleg
betölteni az adatokat.
Nagyon rossz ez a hétvége? Azaz az óránkénti bontás mellett a napi adat is
szar?

Zs.

**Comment by bankizsolt** [comment]
Azt a két napot egyáltalán nem frissítette visszamenőleg. Nincs óránkénti bontás. 
Megoldás lehet visszamenőleg lekérni. Azt hogyan kell? 
![Screenshot_2023-01-20-14-51-25-521_io homeassistant companion android](https://user-images.githubusercontent.com/60253654/213713489-daa94fb8-6397-42d0-9157-17bea7cfa490.jpg)
![Screenshot_2023-01-20-14-51-20-565_io homeassistant companion android](https://user-images.githubusercontent.com/60253654/213713501-7508ca49-3958-429f-8e5d-e86037fd9959.jpg)

**Comment by bankizsolt** [comment]
Kedves Zsombor!
Bármi megoldás a problémára?

**Reply by ZsBT** [reply]
Szia Zsolt!

Átnéztem; szerintem az alábbi szerint lehet lehúzni régebbi adatokat.
1) a portál felületen átállítani a riportban a napok számát ( 3 nap -> 30 nap )
2) a custom_components/w1000-energy-monitor/__init__.py fájlban a 205. sorban a 96*2 -t átírni 96*30-ra. Ez határozza meg a betöltendő napok számát.

Home Assistant újraindítás kell hozzá. Betöltés után érdemes még aznap valamikor visszaírni és megint újraindítani, mert az EON nagyon érzékeny arra ha sok adatot húzunk le; rögtön fenyegető emaileket küld :(

Üdv: Zsombor

**Reply by bankizsolt** [reply]
Szia! Köszönöm. Végül tegnap valamiért újra ránéztem a HA fejlesztői eszközök - statisztika menüben azokra a napokra és ott voltak a fals adatok. Nem tudtam az óránkénti adatokat tisztázni, de legalább a napi adatot egyenesre tettem. Mostmár így marad. Remélem nem sűrűn lesz ilyen szerver oldali hiba.

Üdv. 
Zsolt

**Comment by toranis** [comment]
Kedves Zsombor! Nekem is segítségre lenne szükségem a múltbéli adatokkal kapcsolatban. Nem lehet valahogy távleolvasás-portál oldali xls , vagy csv exportból importálni az adatokat? Én most telepítettem és aktiváltam a leírás szerint a repo-dat és a -december-január-február adatok eléggé összekeveredtek. A korábbi teljes termelést és fogyasztást beimportálta, de egy hónapra:
![2023-02-01](https://user-images.githubusercontent.com/95634502/216143218-d6f87bfb-73f1-4125-9cab-891b94822323.png)

**Comment by ZsBT** [comment]
Szia! 
Korábban honnan érkezett HA-ba az adat?
Az az óriási tüske sajnos benne marad, mert az utolsó napra a szenzor értékét, azaz a teljes óraállást jegyzi be napi értéknek.
Ez csúfítja el ilyen nagyon a grafikont, ha nagyobb távlatból nézzük. Az előző nap/hét/hónap esetén, amiben már nincs benne az utolsó nap, rendesen fogja mutatni.
Ennek javításához teljesen át fogom alakítani a komponenst, ami picit kényelmetlen lesz mert mindenkinek hozzá kell nyúlnia az EON portálon a riportokhoz.
Ez a frissítés később várható, mert cefetül ki akarom tesztelni hogy ne törjön el mindenkinél az integráció (amikor azt hiszem hogy nálam jó, akkor másnál is).
Zsombor

**Reply by toranis** [reply]
Korábban csak a termelésről érkezett adat, mert a growatt servert egyből megtaláltam, amikor elkezdtem a bubuckodni a HA-val, de a ti repo-tokat csak most találtam meg. A leírás alapján elindítottam az adatimportot, és ez lett belőle. A napelemeim pedig már 2020 decembere óta fent vannak, így ott volt adat bőven, amit a growatt server szépen szolgáltat is.

**Reply by bvamos** [reply]
Szívesen segítek tesztelni, engem nem zavar, ha esetleg nem pontosak az adataim, de akár egy másik HA környezetet is tudok csinálni hozzá.

**Comment by ZsBT** [comment]
Köszönöm mindenkinek a tapasztalatokat, a tesztelőknek külön örülök.

[Indítottam új szálat](https://github.com/ZsBT/hass-w1000-portal/discussions/15) ami kimondottan erről szól, folytassuk ott a diskurzust.

**Comment by fabizoli** [comment]
Szia!

Szerinted miért nem jelennek meg az adatok?
Mit hagytam ki?

![image](https://github.com/ZsBT/hass-w1000-portal/assets/45947183/f9ddc218-ad24-4c9a-aab2-d14182221b03)

**Reply by ZsBT** [reply]
Úgy látom aug.3 óta nem jönnek a negyedórás adatok :( nálam is tök üres, és a webes portálra lépve se látom. Írok az EONnak...

**Reply by bvamos** [reply]
Nálam sem jön azóta adat

**Reply by fabizoli** [reply]
Köszi

**Reply by ZsBT** [reply]
Ezt válaszolták, szóval majd javul a cucc:

>  (7 August 2023 21:32)
> Tisztelt Partnereink!
> A távleolvasási Portál felületén az adatok jelenleg nem frissülnek egy IT hiba miatt.
> Szakembereink már dolgoznak a hiba elhárításán, a javításig türelmüket és megértésüket kérjük.
> 
> Üdvözlettel:
> Mérési Központ

**Reply by fabizoli** [reply]
Szia Zsolt!

Nekem azóta se jelennek meg a napi adatok... :(
lehet hogy kihagyok valamit?
![image](https://github.com/ZsBT/hass-w1000-portal/assets/45947183/4af8e4d3-c142-4b9a-b7bd-7d7360a092f7)

**Reply by sthevenp** [reply]
Kell egy (number type) helper:
![Screenshot 2024-04-30 at 14 43 36](https://github.com/ZsBT/hass-w1000-portal/assets/30027604/f66f7e24-d1ed-4cef-8ac5-f000f12543b8)

**Reply by fabizoli** [reply]
Igen! Ez hiányzott! Köszi

**Comment by Netesfiu** [comment]
Sziasztok!

A múltbéli adatok importálásához készítettem egy szkriptet, amivel a hiányzó adatok visszatölthetőek a szenzorba, az E.ON-ról exportált adatok alapján, ha telepítve van a Spook integráció: https://github.com/Netesfiu/EON-SPOOKER

![kép](https://github.com/ZsBT/hass-w1000-portal/assets/68812986/e04255f4-0683-4a82-bcbd-33af4b1939d7)

![kép](https://github.com/ZsBT/hass-w1000-portal/assets/68812986/f2dd0274-8014-44fd-83d3-332febf7bdee)

**Reply by ZsBT** [reply]
Ez szuper, köszönjük!
